docs/api.md

```diff
@@ -229,4 +229,7 @@ class MiddlewareRouter:
     def connect(self, path, handler):
     def trace(self, path, handler):
     def any(self, path, handler):
-```
\ No newline at end of file
+```
+
+# Examples
+- [Examples](examples.md)
```

Comment:
Here you can do 
```
### Next [Examples(examples.md)
```

Reply:
do you mean:
# Examples
### Next [Examples](examples.md)
?

Reply:
done.